tests/data/ext/sample-status-invalid-status.json

```diff
@@ -0,0 +1,35 @@
+[
+    {
```

Comment:
we talked about adding some kind of comment (maybe as part of one of the strings) to point out what is wrong with this file

Reply:
Sorry - Missed this change. The status/status key was missing in this case. Will send in an update again.